app/views/main.scala.html

```diff
@@ -12,42 +12,77 @@
         <title>@title</title>
         <link href='//fonts.googleapis.com/css?family=Source+Sans+Pro:300,400,700' rel='stylesheet' type='text/css'>
         <link rel="stylesheet" href="@routes.Assets.at("lib/normalize.css/normalize.css")">
-        <link rel="stylesheet" href="@routes.Assets.at("lib/foundation/css/foundation.min.css")">
         <link rel="stylesheet" href="@routes.Assets.at("css/main.min.css")">
         <script src="@routes.Assets.at("lib/modernizr/modernizr.min.js")"></script>
         <link rel="shortcut icon" type="image/png" href="@routes.Assets.at("images/favicon.png")">
     </head>
     <body>
-        <header class="fw-wrapper">
-            <div class="row">
-                <div class="small-12 columns">
-                    <a href="#" class="logo">
-                        @svg.conductrFullColor()
-                    </a>
+        <div class="off-canvas-wrap" data-offcanvas>
+            <div class="inner-wrap">
+                <!-- Off Canvas Menu -->
+                <div class="left-off-canvas-menu">
                     <nav>
-                        <a href="#">Menu Item</a>
-                        <a href="#">Menu Item</a>
+                        <a href="/conductr">Menu Item</a>
+                        <a href="/conductr">Menu Item</a>
```

Comment:
We want to avoid being ConductR specific on this template. This template will likely be used by other projects also.